code/controllers/subsystems/trade.dm

```diff
@@ -136,47 +142,30 @@ SUBSYSTEM_DEF(trade)
 		qdel(AM)
 	return GLOB.price_cache[path]
 
-/datum/controller/subsystem/trade/proc/get_export_cost(atom/movable/target)
-	. = round(get_cost(target))
-
-/datum/controller/subsystem/trade/proc/get_sell_price(path, datum/trade_station/station)
-	. = round(get_new_cost(path) * station.markdown)
+/datum/controller/subsystem/trade/proc/get_price(atom/movable/target, is_export = FALSE)
+	. = round(get_cost(target, is_export))
 
 /datum/controller/subsystem/trade/proc/get_import_cost(path, datum/trade_station/station)
-	. = get_new_cost(path) ? get_new_cost(path) : 100			// Should solve the issue of items without price tags
-	var/markup = 1.2
-	if(istype(station))
-		markup = station.markup
-	. *= markup
+	. = station?.get_good_price(path)								// get_good_price() gets the custom price of the item, if it exists
+	if(!.)
+		. = get_new_cost(path) ? get_new_cost(path) : 100			// Should solve the issue of items without price tags
```

Comment:
We will have to monitor if there is not an edge case where 100 credits is too much. Guildies are efficient to find that kind of monetary loophole.